api_pos_integration.md

```diff
@@ -0,0 +1,275 @@
+# PoS Integration
+
+This documentation describes the snabble API endpoints related to the
+Point of Sale (PoS) integration.  These endpoints are available on the
+`api` subdomain. See [General API access](api_general.md) for general
+information about api access.
+
+## Operations
+
+* [Get checkout process](#get-checkout-process): `GET /{project}/pos/checkout/id/{id}`
+* [Update checkout process](#put-checkout-process): `PUT /{project}/pos/checkout/id/{id}`
+* [Partially update checkout process](#patch-checkout-process): `PATCH /{project}/pos/checkout/id/{id}`
+
+### Data Model
+
+* [Checkout Process](#checkout-process)
+
+## Workflow
+
+The integration of snabble into an Point of Sale software system (PoS)
+is implemented through a handover of an identifier from the snabble app
+to the PoS through a standard QR-Code. After the handover the PoS can
+fetch and process the checkout. Finally, it updates it with the
+outcome of the process.
+
+The QR-Code presented to the PoS by the user contains a string of the form
```

Comment:
`The QR-Code presented by the user contains the identifier of the checkout process. It looks like this:`